src/config/site.ts

```
@@ -95,7 +98,9 @@ export const siteUtils = {
    * Construit le titre complet de la page
    */
   getPageTitle(pageTitle?: string): string {
-    return pageTitle && pageTitle.trim() ? `${pageTitle} | ${siteConfig.title}` : siteConfig.title;
+    return pageTitle && pageTitle.trim()
+      ? `${pageTitle} | ${siteConfig.title}`
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)